kivy/properties.pyx

```diff
@@ -1438,15 +1442,11 @@ cdef class AliasProperty(Property):
         `getter`: function
             Function to use as a property getter.
         `setter`: function
-            Function to use as a property setter. Callbacks bound to the
-            alias property won't be called when the property is set (e.g.
-            `right = 10`), unless the setter returns `True`.
+            Function to use as a property setter.
         `bind`: list/tuple
-            Properties to observe for changes, as property name strings.
-            Changing values of this properties will dispatch value of the
-            alias property.
+            Properties to observe for changes as property name strings.
```

Comment:
I'm not sure why you removed this either?

Reply:
Same as the answer above. That sentence is partially true as new value will be dispatched only when binded property changes and getter returns new value. 
If getter is doing something like clamping (`min(max(x, min_value), max_value)`) and returns the same value then property won't get dispatched.